chat-components/src/components/chatbutton/ChatButton.tsx

```diff
@@ -23,12 +25,13 @@ function NotificationBubble(props: IChatButtonProps, parentId: string) {
     if (unreadMessageCount !== "0") {
         return (decodeComponentString(props.componentOverrides?.notificationBubble) || 
             <Stack
-                role="alert"
+                aria-live="polite" 
                 styles={notificationBubbleStyles}
+                aria-label = {props.controlProps?.ariaLabelUnreadMessageString?? defaultChatButtonControlProps.ariaLabelUnreadMessageString}
                 className={props.styleProps?.classNames?.notificationBubbleClassName}
                 id={parentId + "-notification-bubble"}>
                 {unreadMessageCount}
-                <span style={HiddenTextStyles}>{props.controlProps?.unreadMessageString}</span>
+                <span style={HiddenTextStyles}>{props.controlProps?.unreadMessageString?? defaultChatButtonControlProps.unreadMessageString}</span>
```

Comment:
?? may not work with empty strings the way it is expected. This can be tested but I think if props.controlProps?.unreadMessageString = "", it will still be applied instead of defaultChatButtonControlProps.unreadMessageString.

A safer way would be something like this:
props.controlProps?.unreadMessageString ? props.controlProps?.unreadMessageString : defaultChatButtonControlProps.unreadMessageString